### ISSUE TITLE
annoying 'already imported' warnings

### ISSUE THREAD

**Issue by boegel** [hpcugent/easybuild-framework#1588]
```
/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc/__init__.py:29:
UserWarning: Module vsc was already imported from
/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc/__init__.pyc,
but /usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_install-0.9.12-py2.6.egg is being added to sys.path

  import pkg_resources

/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc/__init__.py:29:
UserWarning: Module easybuild was already imported from 
/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild/__init__.pyc,
but /usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_easyblocks-2.6.0-py2.6.egg is being added to sys.path

  import pkg_resources
```

**Comment by boegel** [comment]
This warning is harmless, and there are workarounds available (e.g. http://stackoverflow.com/questions/3861336/how-do-you-correct-module-already-loaded-userwarnings-in-python), but we should try and get to the bottom of this and have a 'proper' fix...

The split of vsc-base into vsc-base/vsc-install made things worse, since the warning can now be spit out twice; once for the `vsc` namespace, and once for `easybuild`.

cc @JensTimmerman

**Comment by boegel** [comment]
It looks like this is simply a bug in the `setup.py` script, i.e. we should declare `easybuild` (and `vsc`) as a namespace package, see https://pythonhosted.org/setuptools/setuptools.html#namespace-packages

**Comment by stdweird** [comment]
@boegel can you ask the person who can reproduce this to run
```
python -c 'import sys;print sys.version;import vsc;print vsc.__path__;import easybuild;print easybuild.__path__'
```

**Comment by boegel** [comment]
@stdweird: here it is:

```
[oakley02]$ python -c 'import sys;print sys.version;import vsc;print vsc.__path__;import easybuild;print easybuild.__path__'

2.6.6 (r266:84292, May 22 2015, 08:34:51)

[GCC 4.4.7 20120313 (Red Hat 4.4.7-15)]

/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc/__init__.py:29: UserWarning: Module vsc was already imported from /usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc/__init__.pyc, but /usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_install-0.9.12-py2.6.egg is being added to sys.path

  import pkg_resources

['/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.16-py2.6.egg/vsc', '/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_install-0.9.12-py2.6.egg/vsc']

['/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild', '/usr/local/easybuild/2.6.0/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_easyblocks-2.6.0-py2.6.egg/easybuild']
```

**Comment by boegel** [comment]
should be fixed (for `easybuild`) with #1591

**Comment by boegel** [comment]
#1591 was incomplete, and rolled back in #1592 
new attempt in #1593

**Comment by boegel** [comment]
fixed with #1593, follow-up there
